tools/testgrid/testgrid.go

```diff
@@ -80,9 +80,9 @@ func CreateXMLOutput(ts TestSuite, artifactsDir string) error {
 		return err
 	}
 
-	outputFile := artifactsDir + filename
+	outputFile := artifactsDir + "/" + Filename
```

Comment:
what was broken before this fix?

Reply:
got it. Will the file gets too big if we keep appending?

Reply:
They can be as big as one want. It's the caller responsibility to keep that in mind. But I don't see that as an issue in the infra.